src/Framework/ContaoFramework.php

```diff
@@ -297,6 +297,13 @@ private function initializeFramework()
         $this->includeHelpers();
         $this->includeBasicClasses();
 
+        // Start the session because that used to be the case. For BC reasons
+        // we always have to make sure the session is started because people
+        // are relying on $_SESSION to be present/working.
+        if (null !== ($session = $this->request->getSession())) {
+            $session->start();
```

Comment:
This can only be fixed in Contao 4.4. Contao 4.5 explicitly supports using the front end without a session.

Reply:
As I said. We will forever need this in Contao 4.4 because everybody uses `$_SESSION` directly and it will always remain `null` if we don't start it explicitly here.